src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -1227,6 +1412,32 @@ export class PatientViewPageStore {
         ONCOKB_DEFAULT
     );
 
+    readonly structuralVariantOncoKbData = remoteData<IOncoKbData>(
+        {
+            await: () => [
+                this.oncoKbAnnotatedGenes,
+                this.structuralVariantData,
+                this.clinicalDataForSamples,
+                this.studies,
+            ],
+            invoke: async () => {
+                if (AppConfig.serverConfig.show_oncokb) {
```

Comment:
i would hope that we should check this property outside of this resource and not reference this promise if it was false.    i.e. this promise should not know about an app setting